Makefile

```diff
@@ -205,7 +204,7 @@ run-t4c/client/importer: t4c/client/importer
 	if [ "$(PUSH_IMAGES)" == "1" ]; \
 	then \
 		docker tag "$(DOCKER_PREFIX)$(IMAGENAME):$(DOCKER_TAG)" "$(LOCAL_REGISTRY_NAME):$(LOCAL_REGISTRY_PORT)/$(DOCKER_PREFIX)$(IMAGENAME):$(DOCKER_TAG)"; \
```

Comment:
Consider this line too

Reply:
Thanks for the hint. I rewrote the original commit.